nautobot/core/ui/titles.py

```diff
@@ -0,0 +1,127 @@
+from typing import Literal, Optional
+
+from django.template import Context, Template
+from django.utils.html import strip_tags
+
+DEFAULT_TITLES: dict[str, str] = {
+    "list": "{% format_title_with_saved_view verbose_name_plural|bettertitle %}",
```

Comment:
My opinion is still that the saved-view information should maybe be part of the *page heading* as rendered, but should *not* be part of the *document title*.